spec/features/profile_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+feature "profile" do
+
+  context "authorized user" do
+    let(:user) { FactoryGirl.create(:user) }
+
+    scenario "displays welcome message" do
+      sign_in_as(user)
+      visit user_profile_path(user)
+      expect(page).to have_content("#{user.first_name}'s Profile")
+    end
+
+  end
+
+  context "visitor" do
+
+    scenario 'unauth user gets asked to log in' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.